media-player/onju-voice.yaml

```diff
@@ -0,0 +1,482 @@
+                  - script.execute: reset_led
+                else:
+                  - voice_assistant.start:
+          else:
+            - voice_assistant.stop
+            - delay: 1s
+            - script.execute: reset_led
+            - script.wait: reset_led
+            - voice_assistant.start_continuous:
+            
+  - platform: gpio
+    id: mute_switch
+    pin:
+      number: GPIO38
+      mode: INPUT_PULLUP
+    name: Disable wake word
+    on_press:
+      - script.execute: turn_on_wake_word
+    on_release:
+      - script.execute: turn_off_wake_word
+      
+  - platform: status
+    id: api_connection
+    filters:
+      - delayed_on: 1s
+    on_press:
+      - if:
+          condition:
+            and:
+              - switch.is_on: use_wake_word
+              - binary_sensor.is_off: mute_switch
+          then:
+            - voice_assistant.start_continuous:
+    on_release:
+      - if:
+          condition:
+            and:
+              - switch.is_on: use_wake_word
+              - binary_sensor.is_off: mute_switch
+          then:
+            - voice_assistant.stop:
+
+light:
+  - platform: esp32_rmt_led_strip
+    id: leds
+    pin: GPIO11
+    chipset: SK6812
+    num_leds: 6
+    rgb_order: grb
+    rmt_channel: 0
+    default_transition_length: 0s
+    gamma_correct: 2.8
+  - platform: partition
+    id: left_led
+    segments:
+      - id: leds
+        from: 0
+        to: 0
+    default_transition_length: 100ms
+  - platform: partition
+    id: top_led
+    segments:
+      - id: leds
+        from: 1
+        to: 4
+    default_transition_length: 100ms
+    effects:
+      - pulse:
+          name: pulse
+          transition_length: 250ms
+          update_interval: 250ms
+      - pulse:
+          name: slow_pulse
+          transition_length: 1s
+          update_interval: 2s
+      - addressable_lambda: 
+          name: show_volume
+          update_interval: 50ms
+          lambda: |-
+            int int_volume = int(id(onju_out).volume * 100.0f * it.size());
+            int full_leds = int_volume / 100;
+            int last_brightness = int_volume % 100;
+            int i = 0;
+            for(; i < full_leds; i++) {
+              it[i] = Color::WHITE;
+            }
+            if(i < 4) {
+              it[i++] = Color(0,0,0).fade_to_white(last_brightness*256/100);
+            }
+            for(; i < it.size(); i++) {
+              it[i] = Color::BLACK;
+            }
+  - platform: partition
+    id: right_led
+    segments:
+      - id: leds
+        from: 5
+        to: 5
+    default_transition_length: 100ms
+
+number:
+  - platform: template
+    name: "Touch threshold percentage"
+    id: touch_threshold_percentage
+    update_interval: never
+    entity_category: config
+    initial_value: 1.25
+    min_value: 0.25
+    max_value: 5
+    step: 0.25
+    optimistic: true
+    on_value:
+      then:
+        - lambda: !lambda |-
+            id(thresh_percent) = 0.01 * x;
```

Comment:
Does this need to be exposed by default?
Perhaps add `disabled_by_default: true`

Reply:
Good point